native/07_node-operation/100_migration-guides/04_upgrade-guide-spring-1-0.md

```diff
@@ -140,6 +140,6 @@ To accommodate new Savanna consensus information, a new `finality-data-history`
 Scripts that move or delete the ‘data’ directory need to protect the finalizer safety file, or utilize this option to set another location for the finalizer safety.dat file.
 - `finalizers-dir` - Specifies the directory path for storing voting history. Node Operators may want to specify a directory outside of their nodeos' data directory, and manage this as distinct file. More information in [Guide to Managing Finalizer Keys](../../advanced-topics/managing-finalizer-keys).
 
-### New Vote-Threads Option
-Where there is a block producing node that connects to its peers through an intermediate nodeos, the intermediate nodeos will need to have an integer value greater then for `vote-threads`. The default value for `vote-threads` is 4. When `vote-threads` is not an integer greater then zero votes are not propagated.
-- `vote-threads` - Sets the number of threads to handle voting. The default is sufficient for all know production setups, and the recommendation is to leave this value unchanged.
+### New `vote-threads` Option
+Where there is a block producing node that connects to its peers through an intermediate nodeos, the intermediate nodeos will need to have an integer value greater than 0 for `vote-threads`. The default value for `vote-threads` is 4. When `vote-threads` is not an integer greater than 0, votes are not propagated.
```

Comment:
I know this wasn't part of your change, but I just noticed this: Why do we say that the default value for `vote-threads` is 4? The default value is 0. Node operators are recommended to change it to 4 if they want their node to propagate/receive votes.

Reply:
:man_shrugging: I was only copy editing so I didn't notice. Do you want to add a new commit explaining it better? It's a little tricky since it seems like it defaults differently depending on producer configuration.

Reply:
That might be left over from when it was originally defaulted to 4.

Reply:
Current nodeos help:
```
  --vote-threads arg                    Number of worker threads in vote
                                        processor thread pool. If set to 0,
                                        voting disabled, votes are not
                                        propagatged on P2P network. Defaults to
                                        4 on producer nodes.
```

Reply:
Yep, when I looked at the code originally it defaulted to 4.